src/pages/ProfilePage/ProfilePage.js

```diff
@@ -1,14 +1,29 @@
-import React from 'react'
-import profiles from '../../profiles.json'
-import Profile from '../../components/Profile/Profile'
+import React, { useState, useEffect } from 'react'
+import { db } from '../../Firebase'
 import Headline from '../../components/Headline/Headline'
+import Profile from '../../components/Profile/Profile'
 
 export default function ProfilePage() {
+  const [user, setUser] = useState([])
+
+  useEffect(() => {
+    const profiles = db.collection('users').onSnapshot((snapshot) => {
```

Comment:
Wenn ich es richtig verstehe, speicherst du an dieser Stelle unter profiles die Profile der Freunde ab oder? Wenn das nur ein Teil aller in der App gespeicherten Profile sind würde ich es vielleicht nicht profiles sonder profilesFriends oder irgendwie sowas nennen, damit es deutlich wird, dass nicht alle profile dargestellt werden. Wenn ichs falsch verstanden habe einfach irgnorieren...

Reply:
Ne, ich speichere da alle Profile ab :) Trotzdem vielen Dank!